Source/Data/Models/Agent.swift

```diff
@@ -1,22 +1,13 @@
-import ObjectMapper
+import Foundation
 
-class Agent: Mappable {
-    
-    var id: Int!
-    var name: String!
-    var avatar: String!
-    
-    init() {
-        //
-    }
-    
-    required init?(map: Map) {
-        //
-    }
-    
-    func mapping(map: Map) {
-        self.id <- map["id"]
-        self.name <- map["name"]
-        self.avatar <- map["avatar"]
+struct Agent: Codable, Equatable {
+    var id: Int?
```

Comment:
The properties where `implicitly unwrapped`. Removed this because in some API calls this could result the items to not be available and when accessing them the app does crash;

Reply:
The id, name and avatar are not nullable. So the implicit unwrap didn't cause issues. When there is no agent, the `agent` property will be null.

If you found a case where this is crashing, please let us know so we can verify.

Else: Please adjust the definitions to have the fields not-nullable

Reply:
Hmm, will try to test it again with non nil properties.

Reply:
I've tried to make the Agent object like this: 
```swift
struct Agent: Codable, Equatable {
    var id: Int
    var name: String
    var avatar: String
    
    init(id: Int, name: String, avatar: String) {
        self.id = id
        self.name = name
        self.avatar = avatar
    }
}
```
But when I do run the example app I will get this screen when starting a chat: 
![Screenshot 2024-01-17 at 08 24 27](https://github.com/parley-messaging/ios-library/assets/6486389/eb5911e2-40d8-47ac-8ed0-bdea99de132d)

When I change the implementation to this: 
```swift
struct Agent: Codable, Equatable {
    var id: Int
    var name: String?
    var avatar: String?
    
    init(id: Int, name: String?, avatar: String?) {
        self.id = id
        self.name = name
        self.avatar = avatar
    }
}
```
I'm able to start the test app with chat.

Reply:
Thanks, you are right. Looks like `avatar` can be null but `name` shouldn't be. Can you do a quick test if having only the `avatar` nullable makes it error for you as well? (Since having `name` not nullable works fine for me)

Reply:
No, the example app still does give me the error if name is non optional. Strange that it does work for you. I just opened the example app from the project and started a chat.

Reply:
Nevermind, thanks for all the testing! 
We have now verified how this can happen. Name is nullable indeed.

Reply:
Okay, good to know. Thanks for the explanation!